apps/MiUnlockTool/README.md

```diff
@@ -0,0 +1,3 @@
+# MiUnlock Tool
+A Tool helps you to unlock your xiaomi android phone bootloader without root!
+
```

Comment:
```suggestion
XiaoMiTool is an unofficial tool to manage your XiaoMi smartphone ROM and software easily.
XiaoMiTool will take care of most of the operations you would need to do, from when you connect the device to your PC until the desired ROM or piece of software is installed.
XiaoMiTool V2 can download the latest official ROMs, TWRP, root with magisk, and decide which is the best way to install them on your device.
```

Simplified from the official website.

Reply:
Put the contents of the README.md file in a file called `description`. otherwise pi-apps won't do anything with it.